tests/lib/test_config.py

```diff
@@ -1,8 +1,87 @@
 # pylint: disable=missing-function-docstring disable=missing-module-docstring
 
+from unittest.mock import call
+
+import pytest
+
+from nestor_api.config.config import Configuration
 import nestor_api.lib.config as config
+from nestor_api.lib.errors import AppConfigurationNotFoundError
+import nestor_api.lib.io as io
+
+
+def test_change_environment(mocker):
+    spy = mocker.patch.object(io, "execute")
+
+    config.change_environment("environment", "path/to/config")
+
+    spy.assert_has_calls(
```

Comment:
I'm wondering if you should assert on the return values of the tests on this file, seems like we are only spying the calls. Just for consistency maybe you can add some asserts to the return values?